themes/default/assets/css/bundle.css

```diff
@@ -2251,6 +2251,10 @@ section.cloud-overview div.card-inner div.card-inner-content-1{
   align-items:center
 }
 
+section.cloud-overview div.card-inner div.card-inner-content-1 div.icon{
+  min-width:18px
```

Comment:
this is an unrelated change; but stops this icon from getting cut off on smaller screens